packages/flutter/test/material/flat_button_test.dart

```diff
@@ -705,7 +704,7 @@ void main() {
                 visualDensity: visualDensity,
                 key: key,
                 onPressed: () {},
-                child: useText ? const Text('Text', key: childKey) : Container(key: childKey, width: 100, height: 100, color: const Color(0xffff0000)),
+                child: useText ? const Text('Text') : Container(width: 100, height: 100, color: const Color(0xffff0000)),
```

Comment:
I thought I left a comment here in my previous review, but was this change intentional?

Reply:
And I thought I already reverted those changes.  It's just the result of a bad merge.  I think I've reverted them now.